config/locales/en.yml

```diff
@@ -45,6 +45,7 @@ en:
     reply: "Reply"
     reset: "Reset"
     save: "Save"
+    save_continue: "Save and Continue"
```

Comment:
This value is pulled from `t(:proper)[:navigation][:bottom][:save_and_continue]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/navigation/_footer.html.haml